boutiques_descriptors/cbellum.json

```diff
@@ -126,7 +126,19 @@
       "walltime-estimate": 60
   },
   "custom": {
-      "cbrain:readonly-input-files": true, 
-      "cbrain:author": "Natacha Beck <natacha.beck@mcgill.ca>"
+    "cbrain:readonly-input-files": true, 
+    "cbrain:author": "Natacha Beck <natacha.beck@mcgill.ca>",
+    "cbrain:integrator_modules": {
+      "BoutiquesFileTypeVerifier": {
+        "input_file": [ "NpyFile" ]
+      },
+      "BoutiquesFileNameMatcher": {
+        "input_file": "\\.npy$"
```

Comment:
what if it's a `.npy.gz`  though? Maybe we shoudl make the regex support optional compressors extensions....